### ISSUE TITLE
postinstall.pl is missing one character

### ISSUE THREAD

**Issue by nicovell3** [OCSInventory-NG/UnixAgent#262]
Hi,

First of all, thanks for your good work. We've found an issue with the last agent compilation at CentOS 8.

### General informations
Operating system :  CentOS 8
Perl version : v5.26.3 (default)

### OCS Inventory informations
Unix agent version : 2.6.1

### Problem's description
There is one parenthesis missing on line 165 at file postinst.pl. For me, I fixed it running the following command before the `make install`:

```
sed -i '165s/};/});/' postinst.pl
```

### Inventory log file ( optional )
This is the make output before applying the fix:
```
# make install
Manifying 1 pod document
Manifying 5 pod documents
Appending installation info to /usr/lib64/perl5/perllocal.pod
[ ! -f run-postinst ] || /usr/bin/perl postinst.pl
syntax error at postinst.pl line 165, near "};"
Execution of postinst.pl aborted due to compilation errors.
make: *** [Makefile:1346: install] Error 255
```

Thanks!

**Comment by gillesdubois** [comment]
Hi @nicovell3 ,

Are you sure you are using the latest version ?
Your fix has been merged and should be in 2.6.1

See : https://github.com/OCSInventory-NG/UnixAgent/pull/260/files

Regards,
Gilles Dubois.

**Comment by nicovell3** [comment]
Hi @gillesdubois ,

Yes, I'm sure. I've just confirmed that the file I downloaded yesterday isn't updated:

https://github.com/OCSInventory-NG/UnixAgent/archive/v2.6.1.zip

Anyway, I'll use the .tar.gz version the next time then.

Thanks for your help and best regards!
Nico.
